### PR TITLE
Additional p2p fixes

### PR DESCRIPTION
**Pull request by apconole**
These are some additional fixes when using DPDK for p2p testing.

### CODE REVIEW

File: ovs_performance.py [hunk 1]
```diff
@@ -1519,18 +1519,27 @@ def create_ovs_bridge():
...
-                       "other_config:pmd-rxq-affinity={3} " \
-                       "-- set Interface {1} options:n_rxq={2} " \
-                       "other_config:pmd-rxq-affinity={3} ". \
-                       format(config.physical_interface, config.virtual_interface,
+                       "other_config:pmd-rxq-affinity={3} " . \
+                       format(config.physical_interface,
                               config.pmd_rxq_affinity.count(':'), config.pmd_rxq_affinity)
 
+            if config.virtual_interface:
+                command += "-- set Interface {1} options:n_rxq={2} " \
```
Comment (chaudron) [inline ovs_performance.py]: Should this not be {0}, {1}, {2}?

File: ovs_performance.py [hunk 2]
```diff
@@ -1519,18 +1519,27 @@ def create_ovs_bridge():
...
 
     if dpdk:
-        command += "-- set Interface {0} type=dpdk " \
-                   "-- set Interface {1} type=dpdkvhostuser ". \
-                   format(config.physical_interface, config.virtual_interface)
+        command += "-- set Interface {0} type=dpdk " . \
+                   format(config.physical_interface)
+
+        if config.virtual_interface:
+            command += "-- set Interface {1} type=dpdkvhostuser ". \
```
Comment (chaudron) [inline ovs_performance.py]: Should be {0}

File: ovs_performance.py [hunk 3]
```diff
@@ -1519,18 +1519,27 @@ def create_ovs_bridge():
...
+                       format(config.virtual_interface)
 
         if config.pmd_rxq_affinity is not None:
             command += "-- set Interface {0} options:n_rxq={2} " \
-                       "other_config:pmd-rxq-affinity={3} " \
-                       "-- set Interface {1} options:n_rxq={2} " \
-                       "other_config:pmd-rxq-affinity={3} ". \
-                       format(config.physical_interface, config.virtual_interface,
+                       "other_config:pmd-rxq-affinity={3} " . \
```
Comment (chaudron) [inline ovs_performance.py]: Option numbers are messed up here also...

**Review by chaudron — CHANGES_REQUESTED** [review]
See comments

File: ovs_performance.py (hunk 1 above)
Comment (apconole) [reply ovs_performance.py]: Weird.  I'll go back through everything and check how it's all working.  Actually, I don't know if I ever tested the virtual case again, so I probably messed them all up.

Thanks for the review.

**Comment by apconole** [comment]
Forgot to check in and push up the changes.  I've done so - let me know if anything else is lingering.

**Comment by chaudron** [comment]
Thanks for pushing, applied!

**Pull request merged**